src/CMakeLists.txt

```diff
@@ -21,10 +21,11 @@ if(NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Emscripten"
     )
 endif()
 
-include(cmake/sources.cmake)
-include(cmake/sentry.cmake)
+include(shared/sources.cmake)
+include(apps/vpn/cmake/sources.cmake)
```

Comment:
Why to we have all of these vpn includes inside the src/CMakeLists.txt file? Should there be a vpn specific CMakeLists doing all this?

Reply:
because this PR is just about moving files. I'm not changing the build system as we/I like to.